src/server.cpp

```diff
@@ -1606,27 +1606,50 @@ void Server::SendNodeDef(u16 peer_id,
 	Non-static send methods
 */
 
-void Server::SendInventory(PlayerSAO* playerSAO)
+void Server::SendInventory(PlayerSAO* ofPlayerSAO, const std::string& ofName, u16 to_peer_id)
 {
 	DSTACK(__FUNCTION_NAME);
 
-	UpdateCrafting(playerSAO->getPlayer());
+	UpdateCrafting(ofPlayerSAO->getPlayer());
 
 	/*
 		Serialize it
 	*/
 
-	NetworkPacket pkt(TOCLIENT_INVENTORY, 0, playerSAO->getPeerID());
+	NetworkPacket pkt(TOCLIENT_INVENTORY, 0, to_peer_id);
 
 	std::ostringstream os;
-	playerSAO->getInventory()->serialize(os);
+	ofPlayerSAO->getInventory()->serialize(os);
 
 	std::string s = os.str();
 
-	pkt.putRawString(s.c_str(), s.size());
+	pkt << ofName << s;
+
 	Send(&pkt);
 }
 
+void Server::SendInventory(PlayerSAO* playerSAO)
+{
+	DSTACK(__FUNCTION_NAME);
+
+	SendInventory(playerSAO, "", playerSAO->getPeerID());
+}
+
+bool Server::sendInventory(const std::string &of_name, const std::string &to_name)
```

Comment:
I think this should be approached like the tabs vs space discussion when the file is idented with spaces: one large "fix capitalisation" commit breaks git blame, therefore its changes should be as small as possible. But its not that important. It is more important to get this working at all.